docset/winserver2022-ps/grouppolicy/Get-GPO.md

```diff
@@ -210,36 +238,47 @@ Accept wildcard characters: False
-This domain is typically the domain of the user that is running the session.
-For example, the domain of the user who started the session by opening Windows PowerShell from the Program Files menu, or the domain of a user that is specified in a runas command.
-However, computer startup and shutdown scripts run under the context of the LocalSystem account.
-The LocalSystem account is a built-in local account, and it accesses network resources under the context of the computer account.
-Therefore, when this cmdlet is run from a startup or shutdown script, the default domain is the domain to which the computer is joined.
-
-  Only one domain can be used by an instance of this cmdlet.
-If you pipe a collection of GPO (Microsoft.GroupPolicy.Gpo) objects to this cmdlet, the DomainName property of the first GPO object in the collection specifies the domain for the cmdlet.
-This is because domainname is a built-in alias for the *Domain* parameter, and the *Domain* parameter can take its value by property name from the pipeline.
-A non-terminating error occurs for any GPOs in the collection that are not in this domain.
-If this domain is different from the domain of the user account (for startup or shutdown scripts, the computer account), a trust must exist between the two domains.
+
+- You can use the **Domain** parameter to explicitly specify the domain for this cmdlet.
+
+  If you do not explicitly specify the domain, the cmdlet uses a default domain. The default domain
+  is the domain that is used to access network resources by the security context under which the
+  current session is running. This domain is typically the domain of the user that is running the
+  session. For example, the domain of the user who started the session by opening Windows PowerShell
+  from the Program Files menu, or the domain of a user that is specified in a runas command.
+  However, computer startup and shutdown scripts run under the context of the LocalSystem account.
+  The LocalSystem account is a built-in local account, and it accesses network resources under the
+  context of the computer account. Therefore, when this cmdlet is run from a startup or shutdown
+  script, the default domain is the domain to which the computer is joined.
+
+  Only one domain can be used by an instance of this cmdlet. If you pipe a collection of GPO
+  (Microsoft.GroupPolicy.Gpo) objects to this cmdlet, the DomainName property of the first GPO
+  object in the collection specifies the domain for the cmdlet. This is because **domainname** is a
+  built-in alias for the **Domain** parameter, and the **Domain** parameter can take its value by
+  property name from the pipeline. A non-terminating error occurs for any GPOs in the collection
+  that are not in this domain. If this domain is different from the domain of the user account (for
+  startup or shutdown scripts, the computer account), a trust must exist between the two domains.
```

Comment:
Minor change to make sure that we bold the type and property names:

```suggestion
  Only one domain can be used by an instance of this cmdlet. If you pipe a collection of GPO
  (**Microsoft.GroupPolicy.Gpo**) objects to this cmdlet, the **DomainName** property of the first
  GPO object in the collection specifies the domain for the cmdlet. This is because **domainname**
  is a built-in alias for the **Domain** parameter, and the **Domain** parameter can take its value
  by property name from the pipeline. A non-terminating error occurs for any GPOs in the collection
  that are not in this domain. If this domain is different from the domain of the user account (for
  startup or shutdown scripts, the computer account), a trust must exist between the two domains.
```